README.md

```diff
@@ -2,8 +2,10 @@
    <img src="datawave-readme.png" />
 </p>
 
-[![Apache License][li]][ll] ![Build Status](https://github.com/NationalSecurityAgency/datawave/actions/workflows/tests.yml/badge.svg)
+# My Project
 
+[![Apache License][li]][ll] ![Build Status](https://github.com/NationalSecurityAgency/datawave/actions/workflows/tests.yml/badge.svg)
+![2414](https://github.com/NationalSecurityAgency/datawave/actions/workflows/tests.yml/badge.svg?job=build-and-test-microservices)
```

Comment:
We should probably put the build status icons in a table and label them to indicate which is the overall status and which is the docker compose status.